src/murmulator/murmulator_vga/main.cpp

```diff
@@ -315,9 +312,6 @@ int main(){
  
   if (sdCard0.mount()) {
 
-    // Create folders on the SD card if they are missing
```

Comment:
Is this delete on purpose?

Reply:
No! It's not on purpose, it could be an accident.

Reply:
If you could put it back along with the few lines after it, it would be appreciated